src/relayFeeCalculator/chain-queries/ethereum.ts

```diff
@@ -70,15 +75,20 @@ export const SymbolMapping: { [symbol: string]: { address: string; decimals: num
 export const defaultAverageGas = 116006;
 
 export class EthereumQueries implements QueryInterface {
+  private spokePool: EthereumSpokePool;
+
   constructor(
     public readonly provider: providers.Provider,
```

Comment:
Good point. I've modified each provider to have the `readonly` access modifier instead of just private (this [commit](https://github.com/across-protocol/sdk-v2/pull/64/commits/1e82c4e296ca903ffcaf57ec7377f76ee2c9aa32)). I believe these new modifiers align more closely with master and are now consistent.